hack/release/packaging/utils/make-packages.sh

```diff
@@ -49,32 +49,25 @@ for package_type in "$@"; do
 	    # Current time in Debian changelog format; e.g. Wed, 02
 	    # Mar 2016 14:08:51 +0000.
 	    timestamp=`date "+%a, %d %b %Y %H:%M:%S %z"`
-	    for series in trusty xenial bionic focal; do
+	    for series in trusty xenial bionic focal jammy; do
 		{
+			if ${release}; then
+				changelog_message="* ${NAME} v${version} (from Git commit ${sha})."
+			else
+				changelog_message="* Development snapshot (from Git commit ${sha})."
+			fi
 		    cat <<EOF
 ${PKG_NAME} (${DEB_EPOCH}${debver}-$series) $series; urgency=low
 
-EOF
-		    if ${release}; then
-			cat <<EOF
-  * ${NAME} v${version} (from Git commit ${sha}).
-EOF
-		    else
-			cat <<EOF
-  * Development snapshot (from Git commit ${sha}).
-EOF
-		    fi
-
-		    cat <<EOF
-
- -- Neil Jerram <neil@tigera.io>  ${timestamp}
+  ${changelog_message}
 
+ -- Daniel Fox <dan.fox@tigera.io>  ${timestamp}
 EOF
 		} > debian/changelog
 
 		excludes="${DPKG_EXCL:--I}"
 
-		${DOCKER_RUN_RM} calico-build/${series} dpkg-buildpackage ${excludes} -S -d
+		${DOCKER_RUN_RM} calico-build/${series} dpkg-buildpackage ${excludes} -S -d | ts "[build $series]"
```

Comment:
If I'm understanding this and the manpage correctly, this won't actually add a timestamp to each line of output.  Is that what you intended?